.travis.yml

```diff
@@ -23,6 +23,6 @@ install:
   - if [ "$CXX" == "g++"  ]; then export CXX="g++-4.9"; fi
   - if [ "$CXX" == "clang++"  ]; then export CXX="clang++-3.5"; fi
 script:
-  - cmake . && make && make install
+  - mkdir -p build && cd build/ && cmake .. && make && make install
```

Comment:
We want to build *both* in and out of source. We should first build out of source, then in source. Otherwise in source building could break and travis wouldn't know.